src/test/java/uk/gov/justice/hmpps/prison/util/builders/BuilderHelpers.kt

```diff
@@ -235,8 +235,8 @@ fun TestDataContext.getMovements(bookingId: Long): List<ExternalMovement> = this
 fun TestDataContext.getBedAssignments(bookingId: Long): List<BedAssignmentHistory> =
   this.dataLoader.bedAssignmentHistoriesRepository.findAllByBedAssignmentHistoryPKOffenderBookingId(bookingId)
 
-fun TestDataContext.getCaseNotes(bookingId: Long): List<CaseNote> = webTestClient.get()
-  .uri("/api/bookings/{bookingId}/caseNotes?size=999", bookingId)
+fun TestDataContext.getCaseNotes(offenderNo: String): List<CaseNote> = webTestClient.get()
+  .uri("/api/offenders/{offenderNo}/case-notes/v2?size=999", offenderNo)
```

Comment:
This is the endpoint that is being used instead of the booking one

Reply:
I've literally just added a call to that method 😆